sdk/storage/azblob/README.md

```diff
@@ -302,10 +304,8 @@ addition, you can investigate the raw response of any response object:
 ```golang
 var errResp azcore.HTTPResponse
 resp, err := serviceClient.CreateContainer(context.Background(), "testcontainername", nil)
-if err != nil {
-   if errors.As(err, &errResp) {
-        // do something with errResp.RawResponse()
-   }
+if err != nil && errors.As(err, &errResp) {
+    // do something with errResp.RawResponse()
```

Comment:
@jhendrixMSFT does `var errResp azcore.HTTPResponse` have to be a pointer? For example, in tests: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/data/aztables/client_test.go#L30-L31. I suspect this shows up in other places.

Reply:
It depends.  If the error is returned by pointer, then you need to provide the error var by reference (i.e. pointer-to-pointer) in the call to `errors.As()`.

The sample is out-of-date though as `azcore.HTTPResponse` has been removed.  I'll get this fixed.